tests/testthat/test-meltAssay.R

```diff
@@ -0,0 +1,41 @@
+context("meltAssay")
+test_that("meltAssay", {
+
+    data(GlobalPatterns, package="MicrobiomeExperiment")
+    se <- GlobalPatterns
+    molten_assay <- meltAssay(se,
+                              add_col_data = c("X.SampleID", "Primer"),
+                              abund_values = "counts")
+    expect_equal(class(molten_assay)[1], "tbl_df")
+    expect_equal(class(molten_assay)[2], "tbl")
+    expect_equal(class(molten_assay)[3], "data.frame")
+    expect_equal(colnames(molten_assay)[1], "uTaxaID")
+    expect_equal(colnames(molten_assay)[2], "uSamId")
+    expect_equal(colnames(molten_assay)[3], "Abundance")
+    expect_equal(colnames(molten_assay)[11], "X.SampleID")
+    expect_equal(is.numeric(molten_assay$Abundance), TRUE)
+
+    only_assay <- .melt_assay(se, abund_values = "counts")
+    expect_equal(colnames(only_assay)[1], "uTaxaID")
+    expect_equal(colnames(only_assay)[2], "uSamId")
+    expect_equal(colnames(only_assay)[3], "Abundance")
+    expect_equal(is.numeric(only_assay$Abundance), TRUE)
+
+    assay_taxa <- .add_taxonomic_data_to_molten_assay(se, only_assay)
+    expect_equal(colnames(assay_taxa)[1], "uTaxaID")
```

Comment:
`expect_equal(colnames(assay_taxa),c("uTaxaID","uSamId","Abundance","Kingdom"))` is shorter